specs/commandline/inspect.md

```diff
@@ -24,13 +24,10 @@ Upon successful execution, both the digest of the signed artifact and the digest
         └── <signed artifact>
 ```
 
-> [!NOTE]
-> This command is for inspecting signatures associated with OCI artifacts only. Use `notation blob inspect` command for inspecting signatures associated with arbitrary blobs.
-
```

Comment:
This part was not removed in `v1.1.1`. Please send out a PR to `release-1.1` to fix it.

Reply:
@shizhMSFT Created issue to track it:
https://github.com/notaryproject/notation/issues/1012 @JeyJeyGao